docs/developing-for-pulsar/index.md

```diff
@@ -5,4 +5,10 @@ layout: summary.ejs
 
 Community packages extend, customize, and improve the features of Pulsar. The authors of community packages are part of the amazing community that makes Pulsar such a great editor.
 
-If you’ve already read the [Package guides](/package-guides) section, then you might be ready for a deeper understanding of some of the ideas that were covered in that section. If so, read on.
+A typical package will perform exactly one of the tasks listed below, and occasionally may perform more than one:
+
```

Comment:
The intent of this section is not to say “a package will do one of these five things,” but rather to say “a package is the method of distribution for each of these five kinds of customization.”

I'm trying (and perhaps failing) to communicate that

* my `pulsar-language-svelte` package
* my `vibrant-ink-redux` package
* my `calculator-light-ui` package
* my `symbol-provider-bookmarks` package
* and my `inline-autocompletion-textmate` package

are all distributed the same way even though they do _completely_ different things. It's also unusual for a package to do _more than one_ of these things, even though there's not much enforcement of this in Pulsar.

(For instance: I don't think a single package can function as _both_ a UI theme _and_ a syntax theme, since I think it'll show up in only one of the two theme pickers based on your metadata selection. But _either_ kind of theme package can run code! Its `main` field in `package.json` is respected!)

Anyway, I'll add a different section above this about how much freedom you have when authoring packages.

Reply:
Yeah that sounds awesome to me. I did understand you weren't saying it was a restriction, I just wanted to take the chance to re-iterate the freedom in capabilities.

But if you plan to add it elsewhere, nearby that's fine by me